src/userdev/java/net/minecraftforge/gradle/userdev/DependencyManagementExtension.java

```diff
@@ -51,7 +53,6 @@ public ArtifactRepository getRepository() {
         return repository;
     }
 
-    @SuppressWarnings("unused")
```

Comment:
No need for this change. Please remove it.

Reply:
I am confused now, Lex told me that he didn't want these supressions in there.

Reply:
Ah sorry I read the diff wrong, the moment you sent that I was gonna remove my message because it was my bad haha

Reply:
Ah sorry I read the diff wrong, the moment you sent that I was gonna remove my message because it was my bad haha